src/EnergyPlus/CommandLineInterface.cc

```diff
@@ -149,6 +152,8 @@ namespace CommandLineInterface {
 
         opt.add("", false, 0, 0, "Display version information", "-v", "--version");
 
+        opt.add("1", false, 1, 0, "Multi-thread with N threads; 1 thread with no arg.", "-j", "--jobs");
```

Comment:
I'm adding a little phrase in here to describe the GHE usage so it shows up in the usage() display.